{{cookiecutter.project_slug}}/src/data/dataGenerator.js

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import users from './raw/users';
+import articles from './raw/articles';
+import notifications from './raw/notifications';
+import conversations from './raw/conversations';
+
+const populateArticles = () => {
+  articles.forEach(article => {
+    const userArticle = article;
```

Comment:
we don't need this line, you could just name the above line `userArticle`

Reply:
it was tested

Reply:
I know, I tested my self too. As mentioned is a line of code that it is not necessary and makes the code harder to understand

Reply:
I take that back, think that if you are going to modify `article` it makes sense to create this new variable 👍